pkgs/servers/sql/postgresql/timescaledb/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, postgresql }:
+
+stdenv.mkDerivation rec {
+  name = "timescaledb-${version}";
+  version = "0.5.0";
+
+  buildInputs = [ postgresql ];
+
+  src = fetchFromGitHub {
+    owner = "timescale";
+    repo = "timescaledb";
+    rev = version;
+    sha256 = "01swgjw563c42azxsg55ry7cyiipxkcvfrxmw71jil5dxl3s0fkz";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
Why do you create an empty `bin`?

Reply:
https://github.com/NixOS/nixpkgs/issues/22653